plugins/image-prioritizer/load.php

```diff
@@ -6,7 +6,7 @@
  * Requires at least: 6.5
  * Requires PHP: 7.2
  * Requires Plugins: optimization-detective
- * Version: 0.1.4
+ * Version: 0.1.5-alpha
```

Comment:
why `-alpha`?

Reply:
Because this is what we do in core as well to indicate the current version in the codebase is not stable. I also needed to bump the versions because of the version checks that are new in this PR.